x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/field_types/keyword_type.tsx

```diff
@@ -50,7 +50,7 @@ interface Props {
 export const KeywordType = ({ field }: Props) => {
   return (
     <>
-      <EditFieldSection>
+      <BasicParametersSection>
```

Comment:
I think the field description component is missing here. A keyword could be a `multi-field`. 

I'm not sure if there are other types that fall into this category.